proposal.txt

```diff
@@ -0,0 +1,14 @@
+Emma Ladouceur
+
+Final Project Proposal
+
+I want to make a create your own adventure game. It will be about someone trying to get to class starting from somewhere in the city. Probably Bowling green because its the bottom of Manhattan. From there people can choice things such as Bus or Train and other things in order to get uptown to columbia.
+
+What will I use? 
+
+I will use a lot of if else statements for the user choice and I'll use a structure for the user. When you start you can create a player and that will be for example Player1 and it will be a deftype stuct Player. I'll then use functions so that I'm not running everything in the main, so each decision can be through a function. Rather than put in strings the user will input numbers and it will lead to another portion of the story. I think I'll have like three layers of stories or however much I have time for. 
```

Comment:
Generally sounds good.  Please detail a non-trivial example of how you intend to use arrays/strings.  

I think it will be very interesting to see what you come up with regarding the story :)